armada-ths/ais
# Recruitment image upload

Implemented profile picture uploading (for both recruitment application and explicit people/edit).

Wrote tests for profiles, also cleaned up the profile code somewhat.


To test image uploading:
* login into armada as admin
* make sure there is a current fair, I will assume it's year 2017
* goto localhost:8000/fairs/2017/people/1/edit
* edit what you like, but select a picture where it says: "Picture of yourself"
* save, make sure you can see the picture
* goto  localhost:8000/fairs/2017/recruitment
* make sure you have a valid recruitment period available, you might need to add roles
* make a new or edit an existing application, upload a different picture to the "Picture of yourslef" field
* save
* goto localhost:8000/fairs/2017/people/1 or localhost:8000/fairs/2017/recruitment/1/application/1/interview
* and finally make sure you see the second picture in the profile
